### PR TITLE
Attempt to fix master

### PR DESCRIPTION
**Pull request by tudor-malene**
This PR removes test classes from the driver classpath to make it identical to a normal node.
Also fixes tests that are affected by this change.

### CODE REVIEW

File: node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt [hunk 1]
```diff
@@ -65,6 +66,7 @@ class AttachmentLoadingTests {
         }
     }
 
+    @Ignore("Temporarily ignored as it fails with: java.lang.SecurityException: sealing violation: can't seal package net.corda.nodeapi: already loaded")
```
Comment (shamsasari) [inline node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt]: I think I might have a solution for this

Comment (tudor-malene) [reply node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt]: I hoped you would, that's why I ignored it here :)

Comment (shamsasari) [reply node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt]: Can you try and see if this branch helps: https://github.com/corda/corda/tree/shams-enclosed-cordapp

File: node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt [hunk 2]
```diff
@@ -21,10 +21,12 @@ import net.corda.testing.driver.driver
 import net.corda.testing.node.NotarySpec
 import net.corda.testing.node.internal.cordappsForPackages
 import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.Ignore
 import org.junit.Test
 import java.net.URL
 import java.net.URLClassLoader
 
+@Ignore("Temporarily ignored as it fails with: java.lang.SecurityException: sealing violation: can't seal package net.corda.nodeapi: already loaded")
```
Comment (shamsasari) [inline node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt]: What's the plan for this?

Comment (tudor-malene) [reply node/src/integration-test/kotlin/net/corda/node/services/AttachmentLoadingTests.kt]: I'll try your branch first, and then investigate which jars are clashing.
But first let's get master going.

**Pull request merged**